README.md

```diff
@@ -4,4 +4,19 @@
 
 Aplicação criada nas [lives do Lucas Caton](https://www.lucascaton.com.br/lives).
 
-Assista em: [youtube.com/lucascaton](https://www.youtube.com/lucascaton?sub_confirmation=1)!
\ No newline at end of file
+Assista em: [youtube.com/lucascaton](https://www.youtube.com/lucascaton?sub_confirmation=1)!
+
+### Contribuindo para este projeto
+
+#### Usando Docker
+
+Faça uma cópia do `.env.sample` e configure caso desejado. Execute o seguinte comando:
+
+```
+docker-compose pull &&
+docker-compose run --rm app bundle install &&
+docker-compose run --rm app npm install bulma bulma-tooltip &&
+docker-compose run --rm app rake db:create &&
```

Comment:
```suggestion
docker-compose run --rm app rails db:create db:migrate &&
```

Reply:
Shouldn't these lines be in the `Dockerfile` though? 🤔

Reply:
Not really. Calling these commands in the `Dockerfile` creates an unnecessary link between images. That breaks the principle that an image should be reproducible because it creates a dependency with other images.